freqtrade/configuration/configuration.py

```diff
@@ -68,10 +68,8 @@ def load_config(self) -> Dict[str, Any]:
         config: Config = load_from_files(self.args.get("config", []))
 
         # Load environment variables
-        from freqtrade.commands.arguments import NO_CONF_ALLOWED
-        if self.args.get('command') not in NO_CONF_ALLOWED:
-            env_data = enironment_vars_to_dict()
-            config = deep_merge_dicts(env_data, config)
+        env_data = enironment_vars_to_dict()
+        config = deep_merge_dicts(env_data, config)
```

Comment:
We'll definitely want to keep this conditional.

it's intentionally there (added recently) - protecting some odd scenarios where environment variables mess with commands that shouldn't load a configuration.

Reply:
Should be solved, thanks for the explanation.